string source_file
string target_file
int8 patch_size
int8 iters
---
string ann_file
string annd_file
